lzy-api/src/main/proto/ai/lzy/v1/workflow/workflow.proto

```diff
@@ -25,12 +25,14 @@ message Operation {
 
   string poolSpecName = 8;  // Label to execute operation in
 
+  /* optional */ map<string, string> envVariables = 9;  // Environment variables to run operation with
+
   message SlotDescription {
     string path = 1;
     string storageUri = 2;
   }
 
-  message PythonEnvSpec {  // TODO(artolord) remove this and move to upper level
+  message PythonEnvSpec {
```

Comment:
maybe we should rename it to `PythonRunnerSpec` and rename `env` on line 22 to `runner`